packages/core/test-support/helpers/reorder.js

```diff
@@ -1,17 +1,25 @@
-import { find, findAll, settled } from '@ember/test-helpers';
+import { findAll, settled } from '@ember/test-helpers';
 import drag from './drag';
+import $ from 'jquery';
 
 const OVERSHOOT = 2;
 
+/**
+ *
+ * @param {String} mode
+ * @param {String} itemSelector
+ * @param  {...String|Element} resultSelectors - accepts selectors or element objects
+ */
 export default async function reorder(mode, itemSelector, ...resultSelectors) {
   const promises = resultSelectors.map((selector, targetIndex) => async () => {
     await settled();
-    let element = find(selector);
+
+    let element = $(selector)[0];
```

Comment:
Why did we switch to use jquery?

Reply:
`find` doesn't support `:contains` anymore and we need `:contains`